.pylint.rc

```diff
@@ -0,0 +1,58 @@
+[MASTER]
+jobs=4
+persistent=yes
+suggestion-mode=yes
+unsafe-load-any-extension=no
+
+# Blacklist files or directories (basenames, not paths)
+ignore=
+
+# blacklist files or directories by regex  (basenames, not paths)
+ignore-patterns=
+
+[EXCEPTIONS]
+
+overgeneral-exceptions=Exception,RaidenUnrecoverableError
+
+[BASIC]
+
+bad-names=foo,bar,baz,toto,tutu,tata
```

Comment:
this is the default, could be removed, just left it there because it may be useful